src/Axe.Windows.Telemetry/ExceptionExcluder.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Telemetry
+{
+    /// <summary>
+    /// Code to allow code to run but ensure that and Exceptions will be excluded
```

Comment:
This text needs attention.

Reply:
Revised in iteration 6

---
In reply to: [277694010](https://github.com/Microsoft/accessibility-insights-windows/pull/331#discussion_r277694010) [](ancestors = 277694010)